.env.test

```diff
@@ -1,3 +1,4 @@
 VITE_RC_ENDPOINT=http://localhost:8000
 VITE_RC_ANALYTICS_ENDPOINT=http://localhost:8000
-VITE_ASSETS_ENDPOINT=https://da08ctfrofx1b.cloudfront.net
\ No newline at end of file
+VITE_ASSETS_ENDPOINT=https://da08ctfrofx1b.cloudfront.net
+VITE_ALLOW_TAX_CALCULATION_FF=true
```

Comment:
I think we can add this FF env var to the circle CI pipeline so these tests are run also in the CI

Reply:
The problem is when we build the sdk more than the tests themselves